### ISSUE TITLE
How to scroll a scrollable `<div/>` with Lenis?

### ISSUE THREAD

**Issue by nhtoby311** [studio-freight/react-lenis#26]
I have add Lenis to my whole page like in the docs:
```tsx
<ReactLenis root>
  <App/>
<ReactLenis/>
```
I also have component as a `<div/>` that scrollable. I have added `onWheel={(event) => event.stopPropagation()}` to make it scrollable, but it the normal scroll, not Lenis scroll there. How can I apply Lenis scroll to the component as well?

```tsx
<div
     className={styles.scrollCont}
     onWheel={(event) => event.stopPropagation()}
>
   ...scrollable contents
</div>
```

**Comment by GFier** [comment]
Hi! 

I think this [component](https://github.com/studio-freight/sf-website/blob/main/components/scrollable-box/index.js) could be helpful to achieve what you need.

We are using it for the [studio's current site](https://studiofreight.com/), so you can test it there!

Hope it helps.

**Comment by clementroche** [comment]
`data-lenis-prevent` is the way https://github.com/studio-freight/lenis#use-the-data-lenis-prevent-attribute-on-nested-scroll-elements-in-addition-we-advise-you-to-add-overscroll-behavior-contain-on-this-element

**Comment by nhtoby311** [comment]
Hmm I add `data-lenis-prevent` and also the css part to my scrollable div, but it doesnt have the Lenis scroll effect :/ It seems like `data-lenis-prevent` is just a replacement for `onWheel={(event) => event.stopPropagation()}`, which just enough to make the scroll work, but not have Lenis scroll effects

```jsx
<div
      data-lenis-prevent
      className={styles.scrollCont}
      // onWheel={(event) => event.stopPropagation()}
    >
      ...Scrollable
</div>
```

```css
.scrollCont {
  width: 100%;
  height: 100%;
  overflow-y: auto;
  padding: 10px;
  display: flex;
  flex-direction: column;
  gap: 15px;
  overscroll-behavior: contain;
}
```

**Comment by clementroche** [comment]
Please provide a codesandbox.

**Comment by fransjesky** [comment]
i was having similiar issue where i want to make a scrollable div not affected by lenis. using data-lenis-prevent on the container is working fine for me. thank you for the solution!

**Comment by nhtoby311** [comment]
Sure! Here is the sandbox for the issue: https://codesandbox.io/s/lenis-inner-scroll-issue-ymcvvv?file=/src/App.tsx

**Comment by clementroche** [comment]
Ok it should be fixed in v1.0.20

See the example: https://codesandbox.io/s/lenis-inner-scroll-issue-forked-xkrvt7?file=/src/App.tsx

Please use the latest versions of lenis and react-lenis
